Airgap install (#1657)
spectrocloud/librarium: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/install.md

icon: ""
hide_table_of_contents: false
sidebar_position: 0
toc_max_heading_level: 3
tags: ["vertex", "vmware"]
---

You install Palette VerteX using the Palette Command Line Interface (CLI) that guides you for details to create a configuration file and a three-node enterprise cluster for high availability (HA). You can invoke the Palette CLI on any Linux x86-64 system with the Docker daemon installed and connectivity to the VMware vSphere environment where Palette VerteX will be deployed. 

## Prerequisites

:::caution

> If you are installing an airgap Palette, ensure you complete all the airgap pre-install steps before proceeding with the installation. Refer to the [VMware vSphere Airgap Instructions](../airgap/vmware-vsphere-airgap-instructions.md) guide for more information.

Author: Indeed it should.

:::

## Deployment

> The video below provides a demonstration of the installation wizard and the prompts you will encounter. Take a moment to watch the video before you begin the installation process. Make sure to use values that are appropriate for your environment. Use the **three-dot Menu** in the lower right corner of the video to expand the video to full screen and to change the playback speed.

Reviewer: ```suggestion
The video below demonstrates the installation wizard and the prompts you will encounter. Take a moment to watch the video before you begin the installation process. Make sure to use values that are appropriate for your environment. Use the **three-dot Menu** in the lower right corner of the video to expand the video to full screen and to change the playback speed.
```

<br />

4. Depending on that type of install of Palette you are using, the Spectro Cloud repository URL value will be different.
    - Non-Airgap: `https://saas-repo-fips.console.spectrocloud.com`
    - Airgap: The URL or IP address of your HTTP file server that is hosting the manifest files.

> 5. Enter the repository credentials. The credentials to the public Spectro Cloud repository is provided to your by our support team. Airgap installations, provide the credentials to your private repository. If your HTTP file server has no authentication, provide the username and password as `admin` and `admin` respectively.

Reviewer: ```suggestion
5. Enter the repository credentials. Our support team provides the credentials you need to access the public Spectro Cloud repository. Airgap installations, provide the credentials to your private repository. If your HTTP file server has no authentication, provide the username and password as `admin` and `admin` respectively.
```

  | **Registry Name** | Enter the name of the registry. |
  | **Registry Endpoint** | Enter the registry endpoint. |
  | **Registry Base Path** | Enter the registry base path. |

>   |**Allow Insecure Connection** | Bypasses x509 verification. Type `n` to specify a certificate authority in the follow up prompt. |

Reviewer: ```suggestion
  |**Allow Insecure Connection** | Bypasses x509 verification. Type `n` to specify a certificate authority in the follow-up prompt. |
```
